src/core/text/Text.js

```diff
@@ -744,7 +744,7 @@ export default class Text extends Sprite
 
     set text(text) // eslint-disable-line require-jsdoc
     {
-        text = String(text || ' ');
+        text = String(text === '' || text === null ? ' ' : text);
```

Comment:
Could this be changed to `text == null`? It may require disabling the ESLint strict equality check for that line, but it would make it so that `undefined` is handled the same as `null`.